src/tests/Countries.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Countries from '../components/Country';
+import store from '../redux/store';
+
+test('Check countries component rendered correctly', () => {
+  const tree = render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Countries />
+      </BrowserRouter>
+    </Provider>,
+  );
```

Comment:
- [ ] You only made snapshot testing for the app components, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=the%20home%20page.-,Testing%20requirements,-Create%20unit%20tests), Please also add unit tests for pure functions in your Redux code and integration tests using React Testing Library mocking data fetching and Redux Store connection.